words/words.md

```diff
@@ -695,25 +695,25 @@ Lass uns abschließend zurückkehren zu...
+Teams aus Epidemiologinnen und Epidemiologen und Menschen in der Politik ([links](https://www.americanprogress.org/issues/healthcare/news/2020/04/03/482613/national-state-plan-end-coronavirus-crisis/), [rechts](https://www.aei.org/research-products/report/national-coronavirus-response-a-road-map-to-reopening/ ), und [überparteilich](https://ethics.harvard.edu/covid-roadmap)) haben einen Konsens darüber erzielt, wie wir COVID-19 schlagen können und gleichzeitig unser Leben *und* unsere Freiheiten schützen.
 
 Hier ist die grobe Idee, mit einigen (weniger konsensfähigen) Backup-Plänen:
 
 ![](pics/plan.png)
 
 Was bedeutet das für DICH, jetzt gerade?
 
-**Für jeden:** Respektiere die Ausgangsbeschränkungen, damit wir so schnell wie möglich aus Phase I herauskommen können. Wasche dir weiter die Hände. Stelle deine eigenen Masken her. Lade eine *privatsphäreschützende* App zur Ermittlung von Kontaktpersonen herunter, sobald diese im nächsten Monat verfügbar ist. Bleib gesund - körperlich und geistig! Und schreib den lokalen Entscheidungsträgern, dass sie ihren Hintern hochbewegen sollen und...
+**Für jede und jeden:** Respektiere die Ausgangsbeschränkungen, damit wir so schnell wie möglich aus Phase I herauskommen können. Wasche dir weiter die Hände. Stelle deine eigenen Masken her. Lade eine *privatsphäreschützende* App zur Ermittlung von Kontaktpersonen herunter, sobald diese im nächsten Monat verfügbar sind. Bleib gesund - körperlich und geistig! Und schreib den lokalen Entscheidungstragenden, dass sie ihren Hintern hochkriegen sollen und...
 
-**Für Politiker\*innen:** Machen Sie Gesetze zur Unterstützung von Menschen, die sich selbst isolieren/quarantänisieren müssen. Stellen Sie mehr menschliche Kontaktverfolger\*innen ein, *unterstützt* durch datenschutzfreundliche Apps. Leiten Sie mehr Mittel in die Dinge, die wir aufbauen sollten, wie..
+**Für Politikerinnen und Politiker:** Machen Sie Gesetze zur Unterstützung von Menschen, die sich selbst isolieren/quarantänisieren müssen. Stellen Sie mehr menschliche Kontaktverfolgerinnen und Kontaktverfolger ein, *unterstützt* durch datenschutzfreundliche Apps. Leiten Sie mehr Mittel in die Dinge, die wir aufbauen sollten, wie..
 
-**Für Machende:** Baut Tests. Baut Beatmungsgeräte. Baut persönliche Schutzausrüstung für Krankenhäuser und Praxen. Baut Tests. Baut Masken. Baut Apps. Baut antivirale Medikamente, Prophylaktika und andere Behandlungen, die keine Impfstoffe sind. Baut Impfstoffe. Baut Tests. Baut Tests. Baut Tests. Baut Hoffnung.
+**Für Machende:** Baut Tests. Baut Beatmungsgeräte. Baut Personenschutzausrüstung für Krankenhäuser und Praxen. Baut Tests. Baut Masken. Baut Apps. Baut antivirale Medikamente, Prophylaktika und andere Behandlungen, die keine Impfstoffe sind. Baut Impfstoffe. Baut Tests. Baut Tests. Baut Tests. Baut Hoffnung.
```

Comment:
Ich denke, "Maker" sollte nicht ins Deutsche übersetzt werden, weil das absolut unüblich ist. Aller Personen, die sich davon angesprochen fühlen sollen, werden den Begriff "Maker" kennen, und falls Zweifel bestehen, dass andere ihn nicht kennen, könnte man ggf. [eine Erklärung](https://de.wikipedia.org/wiki/Maker) verlinken.

Reply:
Ja, Lena, stimme ich dir zu.

Reply:
Sehe ich auch so.

Reply:
Wie wäre es mit "Herstellerinnen und Hersteller"? Schließlich geht es um Dinge die hergestellt werden und der angesprochene Personenkreis würde sich wohl auch mehrheitlch selbst so nennen

Reply:
Außerdem und in diesem Zusammenhang: "Behandlungsmethoden" statt "Behandlungen"?